airbyte-integrations/connectors/source-rki-covid/source_rki_covid/source.py

```diff
@@ -65,6 +83,27 @@ def path(
         return "germany/age-groups"
 
 
+# class that contains main source states | full-refresh
+class GermanyStatesAgeGroups(RkiCovidStream):
+    """Docs: https://api.corona-zahlen.org/states/age-groups"""
+
+    primary_key = None
+
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
+        if response.json():
+            for key, value in response.json().get("data").items():
+                record = {"abbreviation": key}
+                for grp, data in value.items():
+                    record.update({grp: data})
+                yield record
```

Comment:
Small suggestion that should be equivalent

```suggestion
            for key, record in response.json().get("data").items():
                record["abbreviation"] = key
                yield record
```

Reply:
Thanks @cptjacky, I will look into those.

Reply:
Hi @cptjacky. Can I fix this last `optional change` `later` when there is time. As the current change is very important to us and our clients and we require the `state data` streams that I have added.

Reply:
Yes we can leave this like this!

However, as mentioned in Saj's comment, please remove all the formatting changes affecting other connectors from your PR before we can merge it!